website/content/docs/concepts/tokens.mdx

```diff
@@ -6,7 +6,7 @@ description: Tokens are a core auth method in Vault. Concepts and important feat
 
 # Tokens
 
-!> **Warning**: Since tokens are considered opaque values, their structure is undocumented and subject to change. For these reasons, we do not recommend using tokens within your scripts or for automation since it can cause breakage.
+!> **Warning**: Since tokens are considered opaque values, their structure is undocumented and subject to change. For these reasons, we do not recommend relying on the structure of tokens within your scripts or automations since it can cause breakage.
```

Comment:
```suggestion
!> **Warning**: Since tokens are considered opaque values, their structure is undocumented and subject to change. For these reasons, we do not recommend relying on the structure of tokens within scripts or automation since it can cause breakage.
```

Reply:
Sorry for the pedantry, but how do you feel about these minor tweaks to the wording?

Reply:
```suggestion
<Warning heading="Internal token structure is volatile">
  Tokens are opaque values so their structure is undocumented and subject to change.
  Scripts and automations that rely on the internal structure of a token in scripts will break.
</Warning>
```

I suggest making the text more direct. Also, the `!>` notation for warnings is deprecated in favor of the tag notation.